client/views/omnichannel/analytics/InterchangeableChart.js

```diff
@@ -24,11 +24,13 @@ const InterchangeableChart = ({ departmentId, dateRange, chartName, ...props })
 
 	const draw = useMutableCallback(async (params) => {
 		try {
-			const result = await loadData(params);
-			if (!(result && result.chartLabel && result.dataLabels && result.dataPoints)) {
-				return console.log('livechat:getAnalyticsChartData => Missing Data');
+			if (params.daterange.from && params.daterange.to) {
+				const result = await loadData(params);
+				if (!(result && result.chartLabel && result.dataLabels && result.dataPoints)) {
+					return console.log('livechat:getAnalyticsChartData => Missing Data');
```

Comment:
There is no reason to display this message on the console log, if something went wrong we should alert the user via notification.